src/index.html

```diff
@@ -1,17 +1,160 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta 
+     name="viewport"
+     content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
+
   <body>
-    <h1>HTML Form</h1>
+    <form
+     class="form" 
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+     method="post"
+    >
+      <fieldset class="form__personal">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input 
+           type="text" 
+           name="surname" 
+           autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          Name:
+          <input 
+           type="text" 
+           name="name" 
+           autocomplete="off"
+          >
+        </label>
+        <label class="form__field">
+          How old are You?
+          <input 
+           type="number" 
+           name="age" 
+           min="1" 
+           max="100" 
+           value="12"
+          >
+        </label>
+        <label class="form__field">
+          Full date of birth:
+          <input 
+           type="date" 
+           name="dateOfBirth"
+          >
+        </label>
+        <label class="form__field">
+          I accept the term of the agreement
+          <input 
+           type="checkbox" 
+           name="termOfAgreement"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form__registration">
+        <legend>Registration</legend>
+        <label class="form__field">
+          E-mail:
+          <input 
+           type="email" 
+           name="email" 
+           placeholder="email@example.com"
+          >
+        </label>
+        <label class="form__field">
+          Password:
+          <input 
+           type="text" 
```

Comment:
```suggestion
           type="password" 
```
lets hide the password

Reply:
> lets hide the password

i have fixed it

Reply:
https://github.com/mate-academy/layout_html-form/pull/2721#discussion_r1020278304
I can not understand what Ihave to change hear?

Reply:
I have fixed my mistakes